thirdparty/github.go

```diff
@@ -271,16 +296,40 @@ func GetGithubCommits(ctx context.Context, oauthToken, owner, repo, ref string,
 	commits, resp, err := client.Repositories.ListCommits(ctx, owner, repo, &options)
 	if resp != nil {
 		defer resp.Body.Close()
+		if err == nil {
+			return commits, resp.NextPage, nil
+		}
+	}
+
+	// fallback to not using github app on error
+	grip.Debug(message.Fields{
+		"ticket":  "EVG-19966",
+		"message": "failed to get commits from GitHub",
+		"caller":  "GetGithubCommits",
+		"owner":   owner,
+		"repo":    repo,
+		"options": options,
+		"resp":    resp,
+		"error":   err.Error(),
```

Comment:
Rather than put the error string in the message fields directly, this should be `grip.Debug(message.WrapError(err, message.Fields{ ... }))`